src/main/java/seedu/address/logic/commands/AddContactCommand.java

```diff
@@ -7,16 +7,15 @@
 import static seedu.address.logic.parser.CliSyntax.PREFIX_ROLE;
 import static seedu.address.logic.parser.CliSyntax.PREFIX_SKILL;
 
-import seedu.address.commons.util.ToStringBuilder;
 import seedu.address.logic.commands.exceptions.CommandException;
 import seedu.address.model.Model;
 import seedu.address.model.person.Person;
 
 /**
  * Adds a person to the address book.
  */
-public class AddContactCommand extends Command {
-    public static final String COMMAND_WORD = "add_contact";
+public class AddContactCommand extends AddCommand<Person> {
+    public static final String COMMAND_ENTITY = "contact";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a contact to the address book. "
             + "Parameters: "
```

Comment:
I think you might have missed updating the MESSAGE_USAGE to COMMAND_WORD + " " + COMMAND_ENTITY

Reply:
Yup good catch